app/src/main/java/com/example/moodtrackerproject/ui/login/LoginFragmentViewModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.moodtrackerproject.ui.login
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+
+internal class LoginFragmentViewModel(app: Application) : AndroidViewModel(app) {
```

Comment:
Use common `ViewModel ` instead of `AndroidViewModel `. We don't need app context here